tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -85,6 +89,8 @@ describe('card_fob_controller', () => {
     timeSelection: TimeSelection;
     showExtendedLine?: Boolean;
     steps?: number[];
+    isProspectiveFobFeatureEnabled?: Boolean;
+    prospectiveStep?: number | null;
```

Comment:
nitpick: Why make this null rather than `undefined`?
```suggestion
    prospectiveStep?: number;
```

Reply:
Different from the code, the test here makes sense to just use undefined, since it will be assigned null in line 144.

Reply:
Ah my bad, I still need null here since I want to test element rendeing on step being null